genie-agent/src/test/groovy/com/netflix/genie/agent/execution/exceptions/AgentExecutionExceptionsSpec.groovy

```diff
@@ -27,31 +27,32 @@ class AgentExecutionExceptionsSpec extends Specification {
     final Throwable CAUSE = new IOException()
 
     @Unroll
-    def "Constructors for exception class #aClass"(Class<? extends Throwable> aClass) {
+    def "Constructors for exception class #aClass"(Class<? extends Throwable> aClass, boolean checked) {
```

Comment:
Just an FYI that you don't need to pass the arguments in the `()` if you don't want to